src/main/java/uk/gov/companieshouse/api/strikeoffobjections/config/ValidationConfig.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.companieshouse.api.strikeoffobjections.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+import uk.gov.companieshouse.api.strikeoffobjections.model.entity.ObjectionStatus;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.AllowedValuesValidationRule;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.DisallowedValuesValidationRule;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.ValidationRule;
+
+import javax.annotation.PostConstruct;
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+public class ValidationConfig {
+
+    private static final String CONFIG_MESSAGE_PREFIX = "CHS ENV CONFIG -";
+
+    @Value("#{'${ACTION_CODES_COMPANY_STRUCK_OFF}'.split(',')}")
+    private List<Long> companyStruckOffActionCodes;
+
+    @Value("#{'${ACTION_CODES_STRIKE_OFF_NOTICE}'.split(',')}")
+    private List<Long> strikeOffNoticeActionCodes;
+
+    @Autowired
+    private ApiLogger apiLogger;
+
+    @Bean
+    public List<ValidationRule<Long>> getActionCodeValidationRules() {
```

Comment:
Very minor but this may be better as a Set rather than a list? Unless we need them ordered? Happy to aprove regardless

Reply:
we do need them ordered, as we may need one rule to run before others. There is also a test for the ordering in the unit test.